src/stream/src/executor/backfill/cdc_backfill.rs

```diff
@@ -485,6 +491,11 @@ impl<S: StateStore> CdcBackfillExecutor<S> {
         cdc_split: &mut Option<SplitImpl>,
         last_binlog_offset: Option<CdcOffset>,
     ) -> StreamExecutorResult<()> {
+        assert!(
+            last_binlog_offset.is_some(),
+            "last binlog offset cannot be None"
+        );
```

Comment:
report error rather than panic here? I think throw an error may interrupt the DDL?

Reply:
> throw an error may interrupt the DDL

This is a good suggestion, this pr is a quick fix for this issue. I'd like to revisit this part after v1.3 release.